files/ansible/library/test_result_parser.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: test_result_parser
+
+short_description: This module parses test output with regexes
+and puts the results into a Junit/Xunit XML files.
+
+version_added: "2.4"
+
+description:
+    - "This is my longer description explaining my sample module"
+
+options:
+    test_output_path:
+        description:
+            - This is the path to the file containing the test's output.
```

Comment:
Avoid "this is ..." on help. Go for simpler descriptions like: "file containing the test output". Same below.